### PR TITLE
Cypress test issues - signIn button click, changePassword test faulty DB state, appDrawer button not always visible

### PR DESCRIPTION
**Pull request by DurkoMatko**
Fixing 3 issues:

- Reintroduced changePassword UI test with same password to avoid putting Db to faulty state
- reLogin sign-in click validates page transition by checking URL(pathname)
- conditional test for presence of appDrawer "hamburger" button to ensure navigation in UI works fine

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -4,9 +4,7 @@
     "testUser": "browsertestuser",
     "testUserMail": "browsertestuser@example.com",
     "testPwd": "browsertestuser",
-    "changePwdUserMail": "changePwdUser@example.com",
-    "changePwdPwd": "changePwdPwd",
-    "newPwd": "newPwd"
+    "newPwd": "browsertestuser"
```
Comment (jamescrowley) [inline cypress.json]: Perhaps would make it clearer if in the test we're just using 'testPwd' too? (to at least be 'honest' that it's only testing a no-op password change)

File: cypress/support/commands.js [hunk 2]
```diff
@@ -30,11 +30,26 @@ Cypress.Commands.add("reLogin", (userMail, userPassword) => {
...
   cy.get("div[data-cy=password] input").type(`${userPassword}`);
   cy.get("button[data-cy=signInButton]").click();
+  cy.location("pathname", { timeout: 10000 }).should("not.include", "/signin");
 });
 
 Cypress.Commands.add("openAppDrawer", () => {
-  cy.get("button[data-cy=appDrawerOpener]").click();
-  cy.get("div[data-cy=appDrawerDiv]").should("be.visible");
+  cy.get("button[data-cy=appDrawerOpener]").should("be.visible");
+  cy.get("body").then($body => {
```
Comment (jamescrowley) [inline cypress/support/commands.js]: This is in a commit called "fix: cypress linter" but it's not very clear why. Perhaps in your commit explain why we're doing this and why the drawer isn't always visible?

Comment (DurkoMatko) [reply cypress/support/commands.js]: Yeah, I don't even understand what happened. I've done` git add -all`, then written commit message and pushed...then linter started to complain(maybe already after commit)..some code got pushed and some didn't even get commited and was only staged. Really no idea what happened..but the changes in those commits don't necessarily fit commit messages.

Comment (DurkoMatko) [reply cypress/support/commands.js]: It's totally weird...it seems the first commit is "fix: cypress linter" commit...soo nothing got commited during my first try...dunno why I thought it did :confused:

Comment (DurkoMatko) [reply cypress/support/commands.js]: Done.

File: cypress.json (hunk 1 above)
Comment (DurkoMatko) [reply cypress.json]: Done.

**Pull request merged**